stack/values.yaml

```diff
@@ -113,103 +181,162 @@ global:
       nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
+      ## @param global.ingress.annotations.nginx.ingress.kubernetes.io/proxy-read-timeout Nginx ingress proxy read timeout
       nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
+      ## @param global.ingress.annotations.nginx.ingress.kubernetes.io/affinity Nginx ingress affinity
       nginx.ingress.kubernetes.io/affinity: "cookie"
+      ## @param global.ingress.annotations.nginx.ingress.kubernetes.io/session-cookie-name Nginx ingress session cookie name
       nginx.ingress.kubernetes.io/session-cookie-name: "argus_sticky_session"
+      ## @param global.ingress.annotations.nginx.ingress.kubernetes.io/session-cookie-max-age Nginx ingress session cookie max age
       nginx.ingress.kubernetes.io/session-cookie-max-age: "600"
+    ## @param global.ingress.tls List of ingress TLS configurations
     tls: []
     #  - secretName: chart-example-tls
     #    hosts:
     #      - chart-example.local
+  
   nodeSelector:
+    ## @param global.nodeSelector.kubernetes.io/arch Node selector for architecture
     kubernetes.io/arch: arm64
 
   # Annotations to add to pods
   podAnnotations:
+    ## @param global.podAnnotations.linkerd.io/inject Linkerd injection annotation
     linkerd.io/inject: enabled
+    ## @param global.podAnnotations.config.linkerd.io/skip-outbound-ports Linkerd skip outbound ports annotation
     config.linkerd.io/skip-outbound-ports: 3306,5432,6379,9300,11211,4444,4567,4568,587,25,2525,465,2465
 
+  # Autoscaling configuration
   autoscaling:
+    ## @param global.autoscaling.enabled Enable autoscaling
     enabled: true
+    ## @param global.autoscaling.minReplicas Minimum number of replicas
     minReplicas: 1
+    ## @param global.autoscaling.maxReplicas Maximum number of replicas
     maxReplicas: 10
+    ## @param global.autoscaling.targetCPUUtilizationPercentage Target CPU utilization percentage
     targetCPUUtilizationPercentage: 80
+    ## @param global.autoscaling.targetMemoryUtilizationPercentage Target memory utilization percentage
     targetMemoryUtilizationPercentage: 80
 
-  # Additional volumes on the output Deployment definition.
+  ## @param global.volumes Additional volumes on the output Deployment definition
   volumes: []
   # - name: foo
   #   secret:
   #     secretName: mysecret
   #     optional: false
 
-  # Additional volumeMounts on the output Deployment definition.
+  ## @param global.volumeMounts Additional volume mounts on the output Deployment definition
   volumeMounts: []
   # - name: foo
   #   mountPath: "/etc/foo"
   #   readOnly: true
 
+  ## @param global.tolerations Tolerations for the pod
   tolerations: []
 
+  ## @param global.affinity Affinity for the pod
   affinity: {}
 
+  ## @param global.topologySpreadConstraints Topology spread constraints for the pod
   topologySpreadConstraints: []
 
-  env: []
+  env:
```

Comment:
nice

Reply:
i messed around with the `modifiers` a bit and they don't seem to be capable of what we needed here so I figured just adding a kind of dummy entry would be best